README.md

```diff
@@ -20,24 +20,28 @@ This is a patch for happybase to support kerberos when connect to hbase thrift s
 
 The patch now support `python 2.7`, `python 3.5`, `python 3.6`, `python 3.7`.
 
+The project is base on happybase https://github.com/python-happybase/happybase
+
+
 # requirements
 
 In order to use kerberos, the following packages is required:
 
-1. python3-dev
-2. libkrb5-dev
```

Comment:
I'm not sure why you have deleted `libkrb5-dev`. It is needed to make `pykerberos` work.  And `python3-dev` is needed to support Python3.

Reply:
On redhat , there is no `python3-dev`, I should keep yours